deploy-manage/remote-clusters/remote-clusters-api-key.md

```diff
@@ -43,104 +43,29 @@ If you run into any issues, refer to [Troubleshooting](/troubleshoot/elasticsear
 ::::{note}
 If a remote cluster is part of an {{ech}} (ECH) deployment, the remote cluster server is enabled by default and it uses a publicly trusted certificate provided by the platform proxies. Therefore, you can skip the following steps in these instructions:
 
-**On the remote (ECH) cluster:** Skip steps 1-4 (enabling the service, generating certificates, configuring SSL settings, and restarting the cluster), and go directly to step 5 (create an API key).
+**On the remote (ECH) cluster:** Skip steps 1-4 (enabling the service, generating certificates, configuring SSL settings, and restarting the cluster), and go directly to [create an API key](#create-api-key).
 
 **On the local (self-managed) cluster:** Do not add the `xpack.security.remote_cluster_client.ssl.certificate_authorities` setting to the configuration file because ECH uses publicly trusted certificates that don't require custom CA configuration.
 ::::
 
 
 ### On the remote cluster [remote-clusters-security-api-key-remote-action]
 
-1. Enable the remote cluster server on every node of the remote cluster. In [`elasticsearch.yml`](/deploy-manage/stack-settings.md):
+#### Enable and secure the remote cluster server interface
```

Comment:
this should get a little intro paragraph, and the sub-steps should probably be numbered?